test.cpp

```diff
@@ -11,8 +11,19 @@ Tests for the Math Class
 #include "math_utils.h"
 #include <iostream>
 #include <vector>
+#include <limits>
 
 
+TEST_CASE ("Testing distance") {
+    std::vector<int> p1 = {1, 2, 3};
+    std::vector<int> p2 = {1, 2, 3};
+    REQUIRE(Math::distance(p1, p2) == 0);
+    std::vector<int> p3 = {1, 1, 1};
+    double ans = 2.2360679775 + std::numeric_limits<double>::epsilon();
```

Comment:
Test cases for your method should be exhaustive. Try adding sections that test points in 1D, 2D, etc. Also test for the corner cases, such as when the point vectors are empty or the size of the point vectors don't match.